src/vorta/scheduler.py

```diff
@@ -58,6 +59,26 @@ def __init__(self):
         # connect signals
         self.app.backup_finished_event.connect(lambda res: self.set_timer_for_profile(res['params']['profile_id']))
 
+        # connect to DBus login Manager to receive sleep/resume events
```

Comment:
```suggestion
        # connect to `systemd-logind` to receive sleep/resume events
        # The signal `PrepareForSleep` will be emitted before and after hibernation.
```
More detailed comment so that one doesn't have to look up the documentation of the interface.